### PR TITLE
Update gradle Java to JDK15 for packaging distro tests

### PR DESCRIPTION
**Pull request by jkakavas**

### CODE REVIEW

File: buildSrc/src/main/java/org/elasticsearch/gradle/internal/test/DistroTestPlugin.java [hunk 1]
```diff
@@ -58,8 +58,8 @@
 public class DistroTestPlugin implements Plugin<Project> {
     private static final String SYSTEM_JDK_VERSION = "11.0.2+9";
     private static final String SYSTEM_JDK_VENDOR = "openjdk";
-    private static final String GRADLE_JDK_VERSION = "14+36@076bab302c7b4508975440c56f6cc26a";
-    private static final String GRADLE_JDK_VENDOR = "openjdk";
+    private static final String GRADLE_JDK_VERSION = "15.0.2+7";
```
Comment (jkakavas) [inline buildSrc/src/main/java/org/elasticsearch/gradle/internal/test/DistroTestPlugin.java:61]: I couldn't get the jdkdownloadplugin to download an openjdk 15 one. Assuming adoptopenjdk is also fine

**Comment by jkakavas** [comment]
@elasticmachine update branch

File: buildSrc/src/main/java/org/elasticsearch/gradle/internal/test/DistroTestPlugin.java (hunk 1 above)
Comment (mark-vieira) [reply buildSrc/src/main/java/org/elasticsearch/gradle/internal/test/DistroTestPlugin.java:61]: adoptopenjkd should be fine.

**Review by mark-vieira — APPROVED** [review]
:+1:

**Pull request merged**